<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPrefSoundDlg</class>
 <widget class="QWidget" name="DlgPrefSoundDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>363</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="settingsVLayout">
     <item>
      <layout class="QHBoxLayout" name="apiHLayout">
       <item>
        <widget class="QLabel" name="apiLabel">
         <property name="text">
          <string>Sound API</string>
         </property>
         <property name="buddy">
          <cstring>apiComboBox</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="apiComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="sampleRateHLayout">
       <item>
        <widget class="QLabel" name="sampleRateLabel">
         <property name="text">
          <string>Sample Rate</string>
         </property>
         <property name="buddy">
          <cstring>sampleRateComboBox</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="sampleRateComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="audioBufferHLayout">
       <item>
        <widget class="QLabel" name="audioBufferLabel">
         <property name="text">
          <string>Audio Buffer</string>
         </property>
         <property name="buddy">
          <cstring>audioBufferComboBox</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="audioBufferComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="latencyHLayout">
       <item>
        <widget class="QLabel" name="latencyLabel">
         <property name="text">
          <string>Known Latency</string>
         </property>
         <property name="buddy">
          <cstring>audioBufferComboBox</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="currentLatency">
         <property name="text">
          <string>20 ms</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="underflowHLayout">
       <item>
        <widget class="QLabel" name="underflowLabel">
         <property name="text">
          <string>Buffer Underflow Count</string>
         </property>
         <property name="buddy">
          <cstring>audioBufferComboBox</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="bufferUnderflowCount">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="underflowHLayout"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="latencyTipsGroupBox">
     <property name="title">
      <string>Latency Tips</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="latencyTipsLabel">
        <property name="text">
         <string>&lt;ul&gt;
&lt;li&gt;Enlarge your audio buffer if the underflow counter is increasing or you hear pops during playback&lt;/li&gt;
&lt;li&gt;Downsize your audio buffer to improve Mixxx's responsiveness&lt;/li&gt;
&lt;/ul&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="ioTabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="outputTab">
      <attribute name="title">
       <string>Output</string>
      </attribute>
      <layout class="QVBoxLayout" name="outputVLayout">
       <item>
        <spacer name="outputVSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="inputTab">
      <attribute name="title">
       <string>Input</string>
      </attribute>
      <layout class="QVBoxLayout" name="inputVLayout">
       <item>
        <spacer name="inputVSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsHLayout">
     <item>
      <widget class="QPushButton" name="queryButton">
       <property name="text">
        <string>Query Devices</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>Reset to Defaults</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="buttonsHSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
